src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/verbatim/widgets/TwitterPageObject.java

```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.verbatim.widgets;
```

Comment:
Classifying page objects by functionality  not by syntax terms. Change the verbatim part of the package name to thirdparties or widgets and limit the number of levels from 2 to 1. There is no need to make two subfolders